Makefile

```diff
@@ -30,24 +35,24 @@ setup/gomod:
 .PHONY: setup/travis
 setup/travis:
 	@echo Installing Operator SDK
-	@curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.8.1/operator-sdk-v0.8.1-x86_64-linux-gnu && chmod +x operator-sdk && sudo mv operator-sdk /usr/local/bin/
+	@curl -Lo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/v0.8.1/operator-sdk-v0.8.1-x86_64-linux-gnu && chmod +x operator-sdk && sudo mv operator-sdk /usr/local/bin/
 
 .PHONY: code/run
 code/run:
-	@operator-sdk up local --namespace=${NAMESPACE}
+	@$(OPERATOR_SDK) up local --namespace=${NAMESPACE}
```

Comment:
this recipe is broken, should be something along the lines of ` @$(OPERATOR_SDK) run --local --namespace=${NAMESPACE}` for  <0.15.2